selfservice/strategy/oidc/provider.go

```diff
@@ -15,6 +15,10 @@ type Provider interface {
 	AuthCodeURLOptions(r ider) []oauth2.AuthCodeOption
 }
 
+type APIFlowProvider interface {
+	ClaimsFromIdToken(ctx context.Context, rawIdToken string) (*Claims, error)
```

Comment:
In Go it is common to name abbreviations with upper case:

```suggestion
	ClaimsFromIDToken(ctx context.Context, rawIDToken string) (*Claims, error)
```